test/verify/test_gemm_add.cpp

```diff
@@ -57,4 +57,4 @@ struct test_gemm_add : verify_program<test_gemm_add<DType>>
 
 template struct test_gemm_add<migraphx::shape::float_type>;
 template struct test_gemm_add<migraphx::shape::half_type>;
-// template struct test_gemm_add<migraphx::shape::fp8e4m3fnuz_type>;
+// TODO template struct test_gemm_add<migraphx::shape::fp8e4m3fnuz_type>;
```

Comment:
Make this an issue linked to this PR as well to track the TODO

Reply:
https://github.com/ROCm/AMDMIGraphX/issues/3420